### PR TITLE
Add view function to check auction execution side

### PR DESCRIPTION
**Pull request by haythem96**
# Task: Add view function to check auction execution side and update natspec

## Description

Add view function to be used by bot

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Document update

## How Has This Been Tested

Please describe how to test to verify the changes. Provide instructions so we can reproduce.

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] Added video recordings if it is a UI change

### CODE REVIEW

File: packages/hardhat/test/integration-tests/strategy/crab-test.ts [hunk 1]
```diff
@@ -1,80 +0,0 @@
-import { ethers } from "hardhat"
```
Comment (haythem96) [inline packages/hardhat/test/integration-tests/strategy/crab-test.ts:1]: I added this file by mistake

**Review by antoncoding — APPROVED** [review]
add natspec

**Review by aleone — APPROVED** [review]
lgtm

File: packages/hardhat/contracts/strategy/CrabStrategy.sol [hunk 2]
```diff
@@ -881,7 +926,7 @@ contract CrabStrategy is StrategyBase, StrategyFlashSwap, ReentrancyGuard, Ownab
      * @param _ethDelta ETH delta (amount of ETH in strategy)
      * @param _wSqueethEthPrice WSqueeth/ETH price
      * @param _feeAdjustment the fee adjustment, the amount of ETH owed per wSqueeth minted
-     * @return auction type(sell or buy) and auction initial target hedge in wSqueth
+     * @return auction type(sell or buy) and auction initial target hedge in wSqueeth
```
Comment (alpinechicken) [inline packages/hardhat/contracts/strategy/CrabStrategy.sol:929]: missing space

**Review by alpinechicken — COMMENTED** [review]
lgtm

**Pull request merged**